test/environment_unittest.cc

```diff
@@ -83,7 +95,7 @@ TEST_F(EnvironmentTest, NumericProjectIdFromConfigOldStyleCredentials) {
       "CredentialsFile: '" + credentials_file.FullPath().native() + "'\n"
   ));
   Environment environment(config);
-  EXPECT_EQ("12345", environment.NumericProjectId());
+  EXPECT_EQ("", environment.ProjectId());
```

Comment:
Actually, this breaks Travis, because Travis runs on Google Cloud, so we get a non-empty value from the metadata server. You'll need to do what I did in #148.